### ISSUE TITLE
Very simple xargs commands fail

### ISSUE THREAD

**Issue by tomhonour** [pystardust/ytfzf#557]
```
ytfzf --channel-link=https://youtube.com/c/<> | xargs ytfzf -c invidious-channel
```
or even
```
cat a | xargs ytfzf
```
### Describe the bug
I can only get `echo` to work with xargs for ytfzf. Even programs like `cat` fail.

### To Reproduce
Try any command other than echo which will print a string to stdout, and then xargs it in.

### Expected behavior
ytfzf should accept stdout regardless of it's origin.

### Information
 - OS: Debian 11
 - Terminal: xfce-terminal
 - Ytfzf version: latest
- Output of `ls -l "$(which sh)"` (if your using fish: `ls -l (which sh)`):

```
lrwxrwxrwx 1 root root 4 Jul 17 16:43 /usr/bin/sh -> dash
```

**Comment by Euro20179** [comment]
I cannot replicate this, regaurdless, in your situation you don't need xargs as ytfzf accepts stdin as input

`cat a | ytfzf -`
`ytfzf --channel-link=... | ytfzf -c invidious-channel -`

**Comment by tomhonour** [comment]
> I cannot replicate this, regardless, in your situation you don't need xargs as ytfzf accepts stdin as input
> 
> `cat a | ytfzf -` `ytfzf --channel-link=... | ytfzf -c invidious-channel -`
> 
> What error shows up when you try to run that?

Hi Euro20179. Thanks for the quick reply. I get this error (using the channel `thpsvideos`:
```
rm: cannot remove '/tmp/ytfzf-env': No such file or directory
> https://youtube.com/channel/UCIDuzEnSK30aJn1bSVNaaKg
/videos (pg: 1)e (with https://y.com.sb) channel: https://youtube.com/channel/UCIDuzEnSK30aJn1bSVNaaKg
invidious-channel failed to load website
Nothing was scraped
```
I've also made a video of it available on my site: [https://filmsbytom.com/ytfzf0.mp4](https://filmsbytom.com/ytfzf0.mp4).

**Comment by Euro20179** [comment]
yay I was able to replicate the bug, now I just need to fix it.

**Comment by Euro20179** [comment]
Ok so the problem is that something is adding a `\r` (carrage return) (not even a new line) at the end it could be xargs or ytfzf so i'll have to go hunt for that

**Comment by Euro20179** [comment]
This has been resolved in this commit: ae1cbfd02e4cb271b414cfb8b99a4821ecc6b989 and will be in version `2.4.1`

**Comment by tomhonour** [comment]
Thanks so much
